skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -365,6 +365,40 @@ void MpActor::MpApiDeath(MpActor* killer)
   }
 }
 
+void MpActor::ModifyActorValue(espm::ActorValue av, float value)
```

Comment:
`value` should be renamed to improve readability. `percentageDelta` for example.

Reply:
`ModifyActorValue` I think we should rename to `ModifyActorValuePercentage` to prevent confusion.

since there is `ModActorValue` Papyrus native.